packages/teams-js/src/internal/handlers.ts

```diff
@@ -100,7 +100,8 @@ export function registerHandlerHelper(
   contexts: FrameContexts[],
   registrationHelper?: () => void,
 ): void {
-  ensureInitialized(...contexts);
+  // allow for registration cleanup even when not finished initializing
+  handler && ensureInitialized(...contexts);
```

Comment:
if `handler` is null/undefined (which is the case when you want to clean up previous registration), `ensureInitialized` won't actually be called (because of `&&`).

Reply:
Ah, got it that makes sense